modules/calc/src/main/java/com/opengamma/strata/calc/runner/DerivedCalculationFunctionWrapper.java

```diff
@@ -122,11 +123,13 @@ public FunctionRequirements requirements(
       ReferenceData refData) {
 
     // The caller didn't ask for the derived measure so just return the measures calculated by the delegate
-    if (!measures.contains(derivedFunction.measure())) {
+    Measure derivedMeasure = derivedFunction.measure();
+    if (!measures.contains(derivedMeasure)) {
       return delegate.calculate(target, measures, parameters, marketData, refData);
     }
     // Add the measures required to calculate the derived measure to the measures requested by the caller
-    Set<Measure> requiredMeasures = Sets.union(measures, derivedFunction.requiredMeasures());
+    SetView<Measure> allRequiredMeasures = Sets.union(measures, derivedFunction.requiredMeasures());
```

Comment:
Did you mean to use `SetView` as the type here? That happens to me when I let the IDE choose the type.